modules/weko-admin/weko_admin/cli.py

```diff
@@ -137,3 +137,37 @@ def save_report_target(target_id, target_name, target_unit):
         click.secho('insert report target success')
     except Exception as e:
         click.secho(str(e))
+
+
+@click.group()
+def billing():
+    """Billing commands."""
+
+
+@billing.command('create')
+@click.argument('user_id')
+@click.argument('is_active')
+@with_appcontext
+def add_billing_user(user_id, is_active):
+    """Ex: ja Japanese true 12 true."""
+    try:
+        BillingPermission.create(user_id, is_active)
+        click.secho('insert billing user success')
+    except Exception as e:
+        click.secho(str(e))
+
+
+@billing.command('active')
+@click.argument('user_id')
+@click.argument('is_active')
+@with_appcontext
+def toggle_active_billing_user(user_id, is_active):
+    """Add new user can access billing file."""
```

Comment:
Please add comment for parameter of this method beside currently comment.

Reply:
Thank for your comment.
It had been fix at last rev.